reconcile/terraform_cloudflare_dns.py

```diff
@@ -0,0 +1,292 @@
+import logging
+import sys
+from typing import (
+    Any,
+    Callable,
+    Dict,
+    List,
+    Mapping,
+    Optional,
+    Union,
+)
+
+from reconcile.gql_definitions.terraform_cloudflare_dns import (
+    terraform_cloudflare_zones,
+)
+from reconcile.gql_definitions.terraform_cloudflare_dns.terraform_cloudflare_zones import (
+    AWSAccountV1,
+    CloudflareAccountV1,
+    CloudflareDnsZoneQueryData,
+    CloudflareDnsZoneV1,
+)
+from reconcile.status import ExitCodes
+from reconcile.typed_queries.app_interface_vault_settings import (
+    get_app_interface_vault_settings,
+)
+from reconcile.utils import gql
+from reconcile.utils.defer import defer
+from reconcile.utils.exceptions import SecretIncompleteError
+from reconcile.utils.external_resources import ExternalResourceSpec
+from reconcile.utils.runtime.integration import (
+    DesiredStateShardConfig,
+    QontractReconcileIntegration,
+)
+from reconcile.utils.secret_reader import (
+    SecretReaderBase,
+    create_secret_reader,
+)
+from reconcile.utils.semver_helper import make_semver
+from reconcile.utils.terraform.config_client import TerraformConfigClientCollection
+from reconcile.utils.terraform_client import TerraformClient
+from reconcile.utils.terrascript.cloudflare_client import (
+    DEFAULT_CLOUDFLARE_ACCOUNT_2FA,
+    DEFAULT_CLOUDFLARE_ACCOUNT_TYPE,
+    CloudflareAccountConfig,
+    TerraformS3BackendConfig,
+    TerrascriptCloudflareClient,
+    create_cloudflare_terrascript,
+)
+
+DEFAULT_NAMESPACE: Mapping[str, Any] = {
+    "name": False,
```

Comment:
I mentioned this in a previous review, but why are these `False` instead of `None`? They're not boolean values, they don't have a value.

Reply:
No particular reason. I probably made a mistake. Changing it to None